WindowsServer_2016/job/build.ps1

```diff
@@ -0,0 +1,5 @@
+Function main([string] $scriptPath, [string] $envsPath) {
+  <%= obj.reqExecCommand %> "$scriptPath" "$envsPath"
+}
+
+main @args
```

Comment:
Does this fail with the appropriate exit code if there is a failure?

Reply:
Seems like the overall status for a failing job is not updated for container builds.

![image](https://user-images.githubusercontent.com/4211715/35562934-0c94590e-05db-11e8-8977-8971ab34ce1a.png)

I guess this is due to this. Will check it and fix it in upcoming PR.